components/datetime/src/fields/macros.rs

```diff
@@ -1,138 +0,0 @@
-// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
Nit: It would reduce diff size if you would keep code where it currently lives

Reply:
It took me a while to find where this macro was defined and I want to save the next editor some time.